### ISSUE TITLE
Back again ? periodic FPS drop in cockpit/displays - STUTTER - CoherentGTDraw low

### ISSUE THREAD

**Issue by EisernUnion** [flybywiresim/a32nx#7009]
### Aircraft Version

Experimental

### Build info

```json
{
  "built": "2022-04-03T09:13:28+00:00",
  "ref": "experimental",
  "sha": "ad4482e1ef1d0b2e86ccb988b000ff58c0f7d635",
  "actor": "tracernz",
  "event_name": "manual",
  "pretty_release_name": "experimental:ad4482e1"
}
```

### Describe the bug

Since the last experimental release i got the feeling something was wrong .... watching the "CoherrentGTUIThread" ... constant spikes from 0.9ms up to 30ms ....

Sounds exactly like in this issue

https://github.com/flybywiresim/a32nx/issues/2428

video clip : https://youtu.be/Pb_8cYpD4Ds``

### Expected behavior

no ms-jumping of CoherrentGTUIThread

### Steps to reproduce

Just fly ?

**Comment by EisernUnion** [comment]
And it seems to be related to the ND, FlightPlans or whatever .... watch this video https://youtu.be/aSnWkveNvJ4 when i switch the nd modes

**Comment by Benjozork** [comment]
Did you have the EFB on ?

**Comment by andrejetcat** [comment]
I also notice, that after switching off the EFB, stutters mainly disappear!

**Comment by themiskt** [comment]
I saw the same behavior today in the last build. FPS Drop and Stutters

**Comment by Benjozork** [comment]
please all state which build you are using

**Comment by andrejetcat** [comment]
....but its experimental:

{
  "built": "2022-04-03T09:13:28+00:00",
  "ref": "experimental",
  "sha": "ad4482e1ef1d0b2e86ccb988b000ff58c0f7d635",
  "actor": "tracernz",
  "event_name": "manual",
  "pretty_release_name": "experimental:ad4482e1"
}
 LATEST

**Comment by EisernUnion** [comment]
Yes.... EFB has huge impact

see here  https://youtu.be/tNXjUZ8DKJ8

**Comment by ldrr** [comment]
I can confirm. Did a massive test with turning the EFB on and off multiple times and there is always stuttering with turned on EFB. No matter what page is opened.

**Comment by andrejetcat** [comment]
also see that the fps drop more, when using pushback feature via the EFB.

**Comment by Seth-Dark** [comment]
Same here with Experimental Aircraft version on latest Beta 9 sim build.
Stutters with fps drops and coherrentGTUIThread spikes. Will check if its EFB and confirm

**Comment by xsimtomx** [comment]
> Thank you for the reports and hard work testing everyone. We are aware of an issue with the new EFB. We recommend keeping it off for the time being while you're not using it until we push a fix. As always we don't have an estimate for when that'll happen.

I know it's a different product, but somehow the stutters are also present in the latest 7.4.0 stable update. I noticed it when using the Headwind A330.

**Comment by ErickSharp** [comment]
Issue has been addressed by @frankkopp in #7044
